apps/moretransit/moretransit.star

```diff
@@ -0,0 +1,213 @@
+"""
+Applet: MoreTransit
+Summary: See next transit arrivals
+Description: See next transit arrivals from TransSee. Optimized for NYC Subway and more customizable than the default apps.
+Author: gdcolella
+"""
+
+load("render.star", "render")
+load("http.star", "http")
+load("schema.star", "schema")
+
+MAX_SUBWAYS = 4
+MIN_MINUTES = 10
+
+SAMPLE_TITLE = "Manhattan"
+
+# Takes the format arguments:
+# TransSee Premium ID
+# TransSee Service
+# StopTag
+# Route
+BASE_URL = "http://transsee.ca/publicJSONFeed?command=predictions&premium={}&a={}&s={}&r={}"
+
+# If no configuration is provided, then default to using some subways
+# near DUMBO, Brooklyn.
+sample_config_line1 = "mtasubway:A:A40N,mtasubway:C:A40N"
+sample_config_line2 = "mtasubway:F:F18N"
+
+# The real API has a lot more response data - this is only
+# the fields that we actually read.
+# This data is used if we don't have a premium TransSee key
+# configured.
+sample_response_data = {
+    "A:A40N": {
+        "predictions": [
+            {"color": "0039a6", "stopTitle": "High St", "direction": [{"prediction": [{"minutes": "10"}, {"minutes": "12"}, {"minutes": "16"}]}]},
+        ],
+    },
+    "C:A40N": {
+        "predictions": [
+            {"color": "0039a6", "stopTitle": "High St", "direction": [{"prediction": [{"minutes": "13"}, {"minutes": "15"}, {"minutes": "17"}]}]},
+        ],
+    },
+    "F:F18N": {
+        "predictions": [
+            {"color": "ff6319", "stopTitle": "York St", "direction": [{"prediction": [{"minutes": "13"}, {"minutes": "15"}, {"minutes": "17"}]}]},
+        ],
+    },
+}
+
+def fetch_data(config):
+    arrival_times = []
+
+    raw_station = config.get("station1") or sample_config_line1
+    parsed = [s.split(":") for s in raw_station.split(",")]
+    stations = [parsed]
+
+    if not config.bool("disableStation2"):
+        raw_station2 = config.get("station2") or sample_config_line2
+        stations.append([s.split(":") for s in raw_station2.split(",")])
+
+    for combined_station in stations:
+        station_name = None
+        station_arrivals = []
+        for service, route, station_code in combined_station:
+            premiumKey = config.str("premium")
+            data = None
+
+            if premiumKey == None:
+                data = sample_response_data[route + ":" + station_code]
```

Comment:
Will it be obvious to users that they need to provide a premium key to get real info?